lib/fable/events.ex

```diff
@@ -193,7 +193,19 @@ defmodule Fable.Events do
     empty_last_event_id = dynamic([agg], is_nil(agg.last_event_id))
 
     prev_agg_and_db_last_event_id_match =
-      dynamic([agg], agg.last_event_id == ^prev_agg.last_event_id)
+      case prev_agg.last_event_id do
+        nil ->
+          dynamic(
+            [agg],
+            is_nil(agg.last_event_id) == is_nil(^prev_agg.last_event_id)
```

Comment:
Just `is_nil(agg.last_event_id)` would be enough here I guess. We already know `prev_agg.last_event_id` is nil, so no need to assert that again in the sql query.

Reply:
Totally agree at first, however the resulting query will differ and it results in an error where do not find any rows to update:
```
** (RuntimeError) Update failed:
#Ecto.Query<from f0 in MyApp.Aggregate, where: f0.id == ^"the_id" and (is_nil(f0.last_event_id) or is_nil(f0.last_event_id))>
Got: {0, nil}
```

Reply:
Yeah. I had to change the final query as well in order to simplify up here. Thanks!